static/css/styles.css

```diff
@@ -109,3 +118,68 @@ h3 {
   flex-direction: column;
   row-gap: 16px;
 }
+
+#logout {
```

Comment:
could it be removed?

Reply:
No it can't. Why should it?

Reply:
seems my missing knowledge for css style sheet language... thought it is commented ;-) fine for me